requirements/extra.txt

```diff
@@ -1,10 +1,5 @@
 # extended list of package dependencies to reach full functionality
 
-neptune-client>=0.4.109
-comet-ml>=3.1.12
-mlflow>=1.0.0
-test_tube>=0.7.5
```

Comment:
@Borda these cannot yet be removed, because they are still not mocked, I will try to do them one by one in follow up

Reply:
the extra.txt is used in CI build

Reply:
I think that no the CI install from `pip install .["extra"]` right?

Reply:
if you add loggers.txt to devel, will it install in ci or not?

Reply:
ok, true, I ll revert...

Reply:
thanks